ethereum/blockcreation/src/test/java/tech/pegasys/pantheon/ethereum/blockcreation/EthHashBlockCreatorTest.java

```diff
@@ -136,4 +143,62 @@ public void createMainnetBlock1_fixedDifficulty1() throws IOException {
     // If we weren't setting difficulty to 2^256-1 a difficulty of 1 would have caused a
     // IllegalArgumentException at the previous line, as 2^256 is 33 bytes.
   }
+
+  @Test
+  public void createMainnetBlock1_noReward() {
```

Comment:
Suggest a name that's more specific to what you're testing:
```suggestion
  public void rewardBeneficiary_zeroReward_skipZeroRewardsFalse() {
```

Reply:
Done